src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,65 @@
+package mate.academy.service
+
+import mate.academy.enums.Color
+import mate.academy.enums.FigureTypes
+import mate.academy.model.Circle
+import mate.academy.model.Figure
+import mate.academy.model.IsoscelesTrapezoid
+import mate.academy.model.Rectangle
+import mate.academy.model.RightTriangle
+import mate.academy.model.Square
+import kotlin.random.Random
+
+const val DEFAULT_RADIUS = 10.0
+const val MAX_FIGURE_SIZE = 100.0
+
+class FigureSupplier {
+    private val figureTypes: Array<FigureTypes> = FigureTypes.values()
+    private val defaultColor: String = Color.WHITE.toString();
```

Comment:
The `defaultColor` is assigned using `Color.WHITE.toString()`. Ensure that `Color.WHITE` is a valid string representation or adjust the implementation if `Color` is an enum with a different structure.

Reply:
the "Color" is enum. what structure to take for that better?